ddraw_logger/ddraw/DirectDraw.cpp

```diff
@@ -519,12 +527,16 @@ HRESULT DirectDraw::SetDisplayMode(
 	str << this << " " << __FUNCTION__;
 	str << " " << dwWidth << "x" << dwHeight << " " << dwBPP;
 
+	/* @TheRedDaemon: removed modification, intention is to see what it normally does, or not?
```

Comment:
Instead of commenting out the display mode modification, I would prefer to have a `#define` config, maybe in `common.h`.